common/core/desktop/tests/unit/kmx/021 - options.kmn

```diff
@@ -1,17 +1,17 @@
 c Description: Tests basic option rules
 c keys: [K_A][K_1][K_A][K_0][K_A]
 c expected: no foo.foo.no foo.
-c context: 
+c context:
 
 store(&version) '10.0'
 
-store(foo) '0'
+store(foo21) '0'
```

Comment:
> * tests that set option to use a test specific option to reduce interference between tests

This should not be necessary: the store names must be specific to a keyboard. Keyboards should be able to reuse store names without problem. 

Is this caused by something in the test harness or by an issue in Keyman Core?

Reply:
That was a problem with the test harness. 

You're right that it is not necessary to have different store names. However, for a test I think it's a bad idea to use the same name since it can create interdependencies between the tests. If we want to test that multiple keyboards can use the same store we should create an explicit test for that.

Reply:
You're right. They are independent. Reverted the changes.